Ginger/GingerCoreNETUnitTest/TestResources/CLI/CLI-CustomExistingRunset_OnlyNames.Ginger.AutoRunConfigs.json

```diff
@@ -0,0 +1,65 @@
+{
+	"SolutionLocalPath": "C:\\Users\\menik\\Documents\\MENIK05\\GitHub\\Ginger\\Ginger\\Ginger\\GingerCoreNETUnitTest\\TestResources\\Solutions\\CLI",
+	"Runset": {
+		"Exist": true,
+		"Name": "Calc_Test",
+		"EnvironmentName": "Env1",
+		"RunAnalyzer": false,		
+		"Runners": [
+			{
+				"Name": "Runner 1",
+				"EnvironmentName": "Env2",			
+				"AppAgentMappings": [
+					{
+						"ApplicationName": "app1",
+						"AgentName": "IE",
+					}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected character ('}' (code 125)): was expecting double-quote to start field name](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4930360)